negroni/adapter.go

```diff
@@ -0,0 +1,49 @@
+package negroniadapter
```

Comment:
I like the name adapter, should we make the same change in the gin implementation?

Reply:
Done